operator/config/manager/manager.yaml

```diff
@@ -67,8 +67,8 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.name
-          - name: EXPOSE_KEPTN_METRICS
-            value: "true"
+          - name: OTEL_COLLECTOR_URL
+            value: otel-collector:4317
```

Comment:
why this change?

Reply:
Metrics are now in the metrics operator, otel collector env default should be put there so that it is generated in the value file of the chart